modules/apps/remote-app/remote-app-admin-web/src/main/java/com/liferay/remote/app/admin/web/internal/application/list/CustomElementPortletAdminPanelApp.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.remote.app.admin.web.internal.application.list;
+
+import com.liferay.application.list.BasePanelApp;
+import com.liferay.application.list.PanelApp;
+import com.liferay.application.list.constants.PanelCategoryKeys;
+import com.liferay.portal.kernel.model.Portlet;
+import com.liferay.remote.app.admin.web.internal.constants.RemoteAppAdminPortletKeys;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Iván Zaera Avellón
+ */
+@Component(
+	immediate = true,
+	property = {
+		"panel.app.order:Integer=110",
```

Comment:
Why 110? the default is usually 100, do you need to overwrite something?

Reply:
We already have custom elements app at lower order and since this needs at least one custom element I thought it made sense to show it below.

In any case, we should redesign the UI to make a proper "welcome" screen, where the user can navigate between the different objects we let them configure. Otherwise it would be somewhat difficult to infer the relationships.

Reply:
👍